app/locales/it/messages.po

```diff
@@ -1274,7 +1274,7 @@ msgstr "Tutorials"
 
 #: app/components/footer.tsx
 msgid "footer.version.change-log"
-msgstr "Registro modifiche"
+msgstr ""
 
```

Comment:
Nit: I think all translations strings for the `Changelog` can be removed :)

Reply:
missed that one thanks